MANUAL.txt

```diff
@@ -1542,6 +1542,10 @@ LaTeX variables are used when [creating a PDF].
 `natbiboptions`
 :   list of options for natbib.
 
+`notes2screen`
```

Comment:
Would it make more sense to call this just `notes`?
Or is there also some way to display notes on the same screen in beamer?

Reply:
@jgm I didn't find a best name. notes can be on separate slides.

Maybe we could just add a raw option `shownotes` which is actually just a passthru for `\setbeameroption` ?

Do you have a better idea ? I'm ok with `notes` if you confirm this choice.

Reply:
Maybe for maximum generality, something like

```
$if(beameroptions)$
$for(beameroption)$$beameroption$$sep$,$endfor$
$endif$
```

then you could have in your YAML
```
beameroptions:
- show notes on second screen=top
```

Reply:
@jgm that's perfect ! Just like classoptions. I'll do this.